source/clear-linux/tutorials/kata.rst

```diff
@@ -43,35 +42,55 @@ Configure Docker to use Kata Containers\* by default
    ExecStart=/usr/bin/dockerd -D --add-runtime kata-runtime=/usr/bin/kata-runtime --default-runtime=kata-runtime
    EOF
    
-Restart the Docker\* and Kata Containers\* systemd services
+Restart the Docker and Kata Containers systemd services
 
 .. code-block:: bash
 
    sudo systemctl daemon-reload
    sudo systemctl restart docker
 
-Run Kata Containers\*
+Run Kata Containers
 *******************
 
 .. code-block:: bash
 
    sudo docker run -ti busybox sh
```

Comment:
When I run this command, after following the note below to set up the proxy variables, I get the following error:

Unable to find image 'busybox.sh:latest' locally
docker: Error response from daemon: pull access denied for busybox.sh, repository does not exist or may require 'docker login'.
See 'docker run --help'.

So I'm making progress but I wasn't expecting the requirement to login to docker.  running the 'docker run --help' does not show me an option to login.  There is a 'docker login' option.  If a end user runs into this we need to walk them through it

Reply:
running 'docker login' generates the following:
Warning: failed to get default registry endpoint from daemon (Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get http://%2Fvar%2Frun%2Fdocker.sock/v1.37/info: dial unix /var/run/docker.sock: connect: permission denied). Using system default: https://index.docker.io/v1/
Login with your Docker ID to push and pull images from Docker Hub. If you don't have a Docker ID, head over to https://hub.docker.com to create one.

Reply:
running 'sudo docker login' removes the warning and displays only the last line:
Login with your Docker ID to push and pull images from Docker Hub. If you don't have a Docker ID, head over to https://hub.docker.com to create one.

Reply:
Issue resolved!